mobile/src/components/interviews/interviews-table/components/id-cell/id-cell.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { FC } from 'react';
+
+import { InterviewsUpdateRequestParamsDto } from '~/common/types/types';
+import { Pressable, Text } from '~/components/common/common';
+
+import { styles } from './styles';
+
+type Props = {
+  id: number;
+  onPress: (id: InterviewsUpdateRequestParamsDto) => void;
+};
+
+const IdCell: FC<Props> = ({ id, onPress }) => {
+  return (
+    <Pressable onPress={(): void => onPress({ id })}>
+      <Text style={styles.id}>{`${id}`}</Text>
```

Comment:
Won't it be enough?
```suggestion
      <Text style={styles.id}>{id}</Text>
```

Reply:
I'd is a number  and text component children must be string

Reply:
use toString instead